blockreassurance.php

```diff
@@ -19,339 +19,418 @@
 * needs please refer to http://www.prestashop.com for more information.
 *
 *  @author PrestaShop SA <contact@prestashop.com>
-*  @copyright  2007-2016 PrestaShop SA
-
+*  @copyright  2007-2019 PrestaShop SA
 *  @license    http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
 *  International Registered Trademark & Property of PrestaShop SA
 */
 
-if (!defined('_CAN_LOAD_FILES_')) {
+if (!defined('_PS_VERSION_')) {
     exit;
 }
 
 use PrestaShop\PrestaShop\Core\Module\WidgetInterface;
 
-include_once _PS_MODULE_DIR_.'blockreassurance/reassuranceClass.php';
+require_once _PS_MODULE_DIR_.'/blockreassurance/classes/ReassuranceActivity.php';
 
-class Blockreassurance extends Module implements WidgetInterface
+class blockreassurance extends Module implements WidgetInterface
 {
+    
+    /** @var string */
+    public $name;
+    
+    /** @var string */
+    public $version;
+    
+    /** @var string */
+    public $author;
+    
+    /** @var bool */
+    public $need_instance;
+    
+    /** @var string */
+    public $module_key;
+    
+    /** @var string */
+    public $author_address;
+    
+    /** @var string */
+    public $controller_name;
+    
+    /** @var bool */
+    public $bootstrap;
+    
+    /** @var string */
+    public $displayName;
+    
+    /** @var string */
+    public $description;
+    
+    /** @var string */
+    public $js_path;
+    
+    /** @var string */
+    public $css_path;
+    
+    /** @var string */
+    public $img_path;
+    
+    /** @var string */
+    public $img_path_perso;
+    
+    /** @var string */
+    public $lib_path;
+    
+    /** @var string */
+    public $docs_path;
+    
+    /** @var string */
+    public $logo_path;
+    
+    /** @var string */
+    public $module_path;
+    
+    /** @var string */
+    public $confirmUninstall;
```

Comment:
Warning: this could be mistaken with a boolean value. `$confirmUninstallMessage` is more clear for example

Reply:
https://github.com/PrestaShop/PrestaShop/blob/4dda842c0d384671e16488136310036bf9dfe750/classes/module/Module.php#L67

It's a predefined variable

Reply:
set the same comment